mobile/test/objective-c/EnvoyTestServerInterface.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+
+@interface TestServerInterface : NSObject
```

Comment:
nit: Comments, please for both the interface and the methods.

Reply:
Not seeing comments.

Reply:
added